waspc/data/Generator/templates/server/rollup.config.js

```diff
@@ -0,0 +1,37 @@
+import esbuild from 'rollup-plugin-esbuild'
```

Comment:
Uses Rollup and the `esbuild` plugin (which is fast and that's want we want in dev)